.github/ISSUE_TEMPLATE/00-bug-issue.md

```diff
@@ -0,0 +1,32 @@
+If the bug is related to a specific library below, please raise an issue in the respective repo directly:
+
+[TensorFlow Data Validation Repo](https://github.com/tensorflow/data-validation/issues)
+
+[TensorFlow Model Analysis Repo](https://github.com/tensorflow/model-analysis/issues)
+
+[TensorFlow Transform Repo](https://github.com/tensorflow/transform/issues)
+
+[TensorFlow Serving Repo](https://github.com/tensorflow/serving/issues)
+
+
+**System information**
+- Have I specified the code to reproduce the issue (Yes/No):
+- Environment in which the code is executed (e.g., Local (Linux/MacOS/Windows), Interactive Notebook, Google Cloud, etc):
+- TensorFlow version (you are using):
+- TFX Version:
+- Python version:
+
+
+**Describe the current behavior**
+
+**Describe the expected behavior**
+
+**Standalone code to reproduce the issue**
+Provide a reproducible test case that is the bare minimum necessary to generate
+the problem. If possible, please share a link to Colab/Jupyter/any notebook.
```

Comment:
I feel that the tone here can be improved a bit better. Consider the following:

```
Providing a bare minimum test cast or steps(s) to reproduce the problem will greatly help us to debug the issue.

Reply:
Done